examples/pytorch/rgcn/entity_sample_multi_gpu.py

```diff
@@ -0,0 +1,151 @@
+"""
+Differences compared to tkipf/relation-gcn
+* l2norm applied to all weights
+* remove nodes that won't be touched
+"""
+import argparse
+import gc
+import torch as th
+import torch.nn.functional as F
+import dgl.multiprocessing as mp
+import dgl
+
+from torchmetrics.functional import accuracy
+from torch.nn.parallel import DistributedDataParallel
+
+from entity_utils import load_data
+from entity_sample import init_dataloaders, init_models, train, evaluate
+
+def run(proc_id, n_gpus, n_cpus, args, devices, dataset, queue=None):
+    dev_id = devices[proc_id]
+    g, num_classes, num_rels, target_idx, inv_target, train_idx,\
+        test_idx, labels = dataset
+
+    dist_init_method = 'tcp://{master_ip}:{master_port}'.format(
+        master_ip='127.0.0.1', master_port='12345')
+    backend = 'nccl'
+
+    if not args.dgl_sparse:
+        backend = 'gloo'
```

Comment:
`backend = 'nccl' if args.dgl_sparse else 'gloo'`

Try avoid negation in condition to increase readability.

Reply:
Done.